platform-sdk/swirlds-wiring/src/test/java/com/hedera/wiring/benchmark/WiringBenchmark.java

```diff
@@ -14,21 +14,21 @@
  * limitations under the License.
  */
 
-package com.swirlds.common.wiring.benchmark;
+package com.swirlds.wiring.benchmark;
```

Comment:
I don't know if this benchmark package is intended to benchmark the whole framework or is just providing functionality. But, if the intention is to test and measure the whole solution, we can provide a jmh module and move all benchmarking-related code there.
Just posting the note here and making the question in Slack.